bgpd/bgpd.c

```diff
@@ -2738,6 +2738,9 @@ static struct bgp *bgp_create(as_t *as, const char *name,
 	struct bgp *bgp;
 	afi_t afi;
 	safi_t safi;
+	struct utsname names;
```

Comment:
this should probably (a) be in `lib/` and (b) respect the `hostname` setting in FRR's config